hmy/downloader/downloaders.go

```diff
@@ -0,0 +1,54 @@
+package downloader
+
+import (
+	"github.com/harmony-one/harmony/core"
+	"github.com/harmony-one/harmony/p2p"
+)
+
+// Downloaders is the set of downloaders
+type Downloaders struct {
+	ds map[uint32]*Downloader
```

Comment:
is there race condition on map access?

Reply:
No. Currently, this map is fixed after setup. We will need to add mutex probably for resharding. For now, I think it's good.